src/Microsoft.AspNetCore.SignalR.Protocols.MsgPack/Internal/Protocol/MessagePackHubProtocol.cs

```diff
@@ -322,7 +322,9 @@ private void WriteInvocationMessage(InvocationMessage message, Packer packer)
                 packer.PackString(message.InvocationId);
             }
             packer.PackString(message.Target);
-            packer.PackObject(message.Arguments, SerializationContext);
+            packer.PackArrayHeader(message.Arguments.Length);
+            foreach (var arg in message.Arguments)
```

Comment:
nit: Add braces. Does this change the generated code or is it just a more efficient way to use the API?

Reply:
You mean the generated `byte[]`? No, its the same, we have quite good tests 😄
It seems to be a better way of using the API, I can grab a dottrace of the after (already have the before) and see why it's so much better